locales/fr.json

```diff
@@ -69,14 +69,14 @@
   "config.locale_country_map": "Objet pour mapper le code local de deux lettres au code du pays",
   "config.locales_paths": "Chemin du répertoire de la locale (par rapport à la racine). Les schémas globaux sont acceptés.",
   "config.namespace": "Activer les namespaces. Vérifiez la documentation pour plus de détails.",
-  "config.path_matcher": "Faire correspondre le chemin pour les locales/namespaces personnalisés. Consultez la documentation pour plus de détails.",
-  "config.preferred_delimiter": "Délimiteur recommandé pour les noms de chemins à trous, par défaut \"-\"(tiret)",
-  "config.prompt_translating_source": "Invite à sélectionner la langue source à chaque fois. Si défini sur False, le langage du système sera automatiquement utilisé.",
-  "config.readonly": "Mode \"Lecture uniquement\"",
+  "config.path_matcher": "Faire correspondre le chemin pour les locales/namespaces personnalisés. Consultez la dcoumentation pour plus de détails.",
+  "config.preferred_delimiter": "Délimiteur recommandé pour les noms de chemins à trous, par défaut «-»(tiret)",
+  "config.prompt_translating_source": "Invite à sélectionner la langue source à chaque fois. Si défini sur False, le langage du système sera autmatiquement utilisé.",
+  "config.readonly": "Mode «Lecture uniquement»",
```

Comment:
Should "dcoumentation" be a "documentation"?

Reply:
Yes it should be "documentation".